src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -870,6 +879,14 @@ func (c controller) getProgressingOLMOperators() ([]*models.MonitoredOperator, e
 		c.log.WithError(err).Warningf("Failed to connect to assisted service")
 		return ret, err
 	}
+	if strings.Contains(c.OpenshiftVersion, "4.8") {
+		for _, operator := range operators {
```

Comment:
Why not to extract it to a function? You use the same code twice.

Reply:
Yes, I am planning to extract this part of the code to GetClusterMonitoredOLMOperators, currently running a manual test to verify everything works as expected on both the OCP versions. Will update the PR after that